src/components/viz/Findings.vue

```diff
@@ -177,8 +177,8 @@ export default {
   },
   data() {
     return {
-      clinicalNotes: null
-
+      clinicalNotes: null,
+      note: null
```

Comment:
I was seeing the following error in the console. Hence added a `note` property
@tonydisera 
![Screen Shot 2020-02-07 at 12 54 41 PM](https://user-images.githubusercontent.com/16284713/74061919-5191d080-49aa-11ea-905f-aabe9324c7fd.png)